### PR TITLE
Add converting files to bigWig

### PR DESCRIPTION
**Pull request by donaldcampbelljr**
Uniwig will create bedGraphs and then use these (in tandem with bigTools) to create bigWig files.

### CODE REVIEW

File: gtars/src/uniwig/writing.rs [hunk 1]
```diff
@@ -115,3 +120,91 @@ pub fn write_to_wig_file(
     }
     buf.flush().unwrap();
 }
+
+pub fn write_to_bed_graph_file(
+    counts: &[u32],
+    filename: String,
+    chromname: String,
+    start_position: i32,
+    stepsize: i32,
```
Comment (nleroy917) [inline gtars/src/uniwig/writing.rs:129]: `u32` since these won't ever be negative?

File: gtars/src/uniwig/writing.rs [hunk 2]
```diff
@@ -115,3 +120,91 @@ pub fn write_to_wig_file(
...
+) {
+    let path = std::path::Path::new(&filename).parent().unwrap();
```
Comment (nleroy917) [inline gtars/src/uniwig/writing.rs:131]: Should we catch any potential errors instead of `unwrap()`?

File: gtars/src/uniwig/writing.rs [hunk 3]
```diff
@@ -115,3 +120,91 @@ pub fn write_to_wig_file(
...
+            count
+        )
+        .unwrap();
+        position = position + stepsize;
+    }
+    buf.flush().unwrap();
+}
+
+/// Converts uniwig generated bedGraphs to bigWig files
+pub fn write_bw_files(location: &str, chrom_sizes: &str, num_threads: i32, zoom_level: i32) {
```
Comment (nleroy917) [inline gtars/src/uniwig/writing.rs:159]: What about changing the type of `num_threads` and `zoom_level` to `u32`? They are being converted anyways down there

**Review by nleroy917 — APPROVED** [review]

**Comment by donaldcampbelljr** [comment]
leaving branch open to continue bam work

**Pull request merged**